TeamCode/src/main/java/org/firstinspires/ftc/teamcode/subsystems/claw/Finger.java

```diff
@@ -0,0 +1,43 @@
+package org.firstinspires.ftc.teamcode.subsystems.claw;
+
+import com.qualcomm.robotcore.hardware.Servo;
+
+class Finger {
+
+    private final double fingerOpenPosition;
+    private final double fingerClosePosition;
+    private final Servo fingerServo;
+    private boolean isOpen;
```

Comment:
If you don't initialize the boolean in the constructor, you may call it before it has a value, causing an error

Reply:
good point